_posts/en/newsletters/2025-03-07-newsletter.md

```diff
@@ -0,0 +1,229 @@
+  the attacker to steal nearly the entire value of a channel.
+
+  Morehouse notes that the other LN implementations have all
+  independently discovered and fixed this vulnerability, including as
+  early as 2018 (see [Newsletter #17][news17 cln2000]), but the LN
+  specification doesn't describe the correct behavior (and may even
+  require the incorrect behavior).  He has [opened a PR][bolts #1233] to
+  update the specification.
+
+- **Discussion about Bitcoin Core's priorities:** several blog posts
+  written by Antoine Poinsot about the future of the Bitcoin Core
+  project were linked in a [thread][poinsot pri] on Delving Bitcoin.  In
+  the [first][poinsot pri1] blog post, Poinsot describes the benefits of
+  long-term goal setting and the costs of ad hoc decision making.  In
+  the [second][poinsot pri2] post, he argues that "Bitcoin Core should
+  be a robust backbone for the Bitcoin network, balancing between
+  securing the Bitcoin Core software and implementing new features to
+  strengthen and improve the Bitcoin network."  In the [third][poinsot
+  pri3] post, he recommends splitting the existing project into three
+  projects: a node, a wallet, and a GUI.  This is within reach today
+  thanks to the multiple-year effort of the multiprocess sub-project
+  (see [Newsletter #39][news39 multiprocess] for our first mention of
+  this sub-project in 2019).
+
+  Anthony Towns [questions][towns pri] whether multiprocess would really
+  allow an effective split, as the individual components would remain
+  tightly coupled.  Many changes to one project would require also
+  making changes to the other projects.  However, it would be a clear
+  win to move features that don't currently require a node into a
+  library or tool that can be maintained independently.  He also
+  describes how some people are using nodes today with middleware that
+  makes it easy for users to connect their wallets to their own personal
+  nodes, although this is done using blockchain indexes (basically [a
+  personal block explorer][topic block explorers])---something the
+  Bitcoin Core project has previously rejected including in its node
+  directly.  Finally, he [notes][towns pri2] that "to [him], providing
+  wallet features (mostly) and a GUI (to a lesser extent[...]) is a way
+  of keeping us honest to the principle of bitcoin being usable by a
+  decentralized bunch of hackers, versus being something that you can
+  only really use if you’re a whale or an established corporation
+  willing to make a big investment."
+
+  David Harding expresses [concern][harding pri] that refocusing the
+  main project on just consensus code and P2P relay will make it harder
+  for everyday users to use a full node to validate their own incoming
+  wallet transactions.  He asks Poinsot and other contributors to
+  consider focusing instead on making Bitcoin Core easier for everyday
+  users.  He describes the power of full node validation: those who
```

Comment:
```suggestion
  users.  He describes the power of full node validation: economically-relevant users who
```
Maybe this is pedantic, but operating a full node alone doesn't help to define the consensus rules. It does little good if I run a forgotten full node in my basement that I never interact with. (It does _some_ good because it filters invalid junk from its peers so they don't have to expend bandwidth and CPU to reject the junk themselves. But this is a small effect.)

The full nodes that _economic_ users run matter. They attach _value_ to coins of one chain (fork) over another, and their full nodes enable them tell the difference.

Reply:
I think the rest of the original sentence covered that point in a more general way: "...full nodes that validate the predominate amount of economic activity...".  The generality is required because _you_ don't need to be an economically relevant user if you operate the full node for people who are themselves economically relevant.  For example, if I operate the node that 10,000 highly active lightweight clients connect to, I don't need to perform any transactions myself in order to have significant economic relevance.

Reply:
That's an excellent point; I hadn't considered the use of light clients.